src/index.html

```diff
@@ -15,8 +15,57 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100..900&family=Montserrat+Alternates:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Montserrat:ital@0;1&family=Roboto&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+              <div class="header__top">
+                  <a href="#">
+                      <img
+                          class="logo"
+                          src="/src/images/logo.png"
+                          alt="logo"
+                      >
+                  </a>
```

Comment:
Fix the indents everywhere
```suggestion
    <header class="header">
      <a href="#">
        <img
          class="logo"
          src="/src/images/logo.png"
          alt="logo"
        >
      </a>
```